alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -519,6 +523,12 @@ fn powerline_drawing(character: char, metrics: &Metrics, offset: &Delta<i8>) ->
     // x = (H - 2) / (2 * slope).
     let x_intersection = (height as i32 + 1) / 2 - 1;
 
+    // Don't use built-in font if we'd cut the tip too much, for example when the font is really
+    // narrow.
```

Comment:
I'm surprised this comment isn't wrapped at 80 columns. Is this only done for doc comments?

Reply:
If you want to wrap at `80` you need to change

`comment_width = 100` in `rustfmt.toml` to `80`, we have it at `100`.